sample/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
     id("com.android.application")
     id("kotlin-android")
     id("kotlin-android-extensions")
+    id("com.vanniktech.maven.publish")
```

Comment:
Do we also need this for the sample project?

Reply:
No, we don't want to publish it to maven central.